Lib/ufo2ft/instructionCompiler.py

```diff
@@ -0,0 +1,227 @@
+# -*- coding: utf-8 -*-
+import array
+import logging
+
+from fontTools import ttLib
+from fontTools.pens.hashPointPen import HashPointPen
+from fontTools.ttLib.tables._g_l_y_f import (
+    OVERLAP_COMPOUND,
+    ROUND_XY_TO_GRID,
+    USE_MY_METRICS,
+)
+
+logger = logging.getLogger(__name__)
+
+ufoLibKey = "public.truetype.instructions"
```

Comment:
Add something like `TRUETYPE_INSTRUCTIONS_KEY = "public.truetype.instructions"`, `OBJECT_LIBS_KEY = "public.objectLib"` and `OBJECT_IDENTIFIERS_KEY = "public.objectIdentifiers"` to  constants.py and use them below.

Reply:
Thank you for your suggestions, I have made all changes accordingly.